winvnc/winvnc/vncdesktop.cpp

```diff
@@ -62,7 +64,7 @@ int getinfo(char mytext[1024]);
 const UINT RFB_SCREEN_UPDATE = RegisterWindowMessage("WinVNC.Update.DrawRect");
 const UINT RFB_COPYRECT_UPDATE = RegisterWindowMessage("WinVNC.Update.CopyRect");
 const UINT RFB_MOUSE_UPDATE = RegisterWindowMessage("WinVNC.Update.Mouse");
-const char szDesktopSink[] = "WinVNC desktop sink";
+const char szDesktopSink[] = "UltraVNC Server desktop sink";
 

```

Comment:
Don't change
This name is used by other softwares.
You changed it to the same in vnchook and winvnc ..correct,
But changed it require a version check between both, older hooks will fail with new server and new hook wil fail on old server.  The risk for possible mistakes it to high